src/app/component/quiz/quiz.component.ts

```diff
@@ -199,33 +195,27 @@ export class Quizcomponent implements OnInit, OnDestroy {
           !this.FormArray.controls.at(this.questionindex)?.get('timer')
             ?.disabled
         ) {
-          this.next(this.questionindex);
+          this.nextQuestion(this.questionindex);
         }
       });
   }
 
   answer(questionindex: number, correctOptions: string) {
```

Comment:
Change this function name to like submitAnswer

Reply:
Yes i changed the function name to submitAnswer